src/codecs/Reaction.ts

```diff
@@ -0,0 +1,60 @@
+import { ContentTypeId, ContentCodec, EncodedContent } from '../MessageContent'
+
+// xmtp.org/text
+//
+// This content type is used for a plain text content represented by a simple string
+export const ContentTypeReaction = new ContentTypeId({
+  authorityId: 'xmtp.org',
+  typeId: 'reaction',
+  versionMajor: 1,
+  versionMinor: 0,
+})
+
+export type Reaction = {
```

Comment:
wonder if we should be namespacing things like this to avoid clashes in the future. maybe something like `ContentTypeReactionContent` or similar?